client/overview/task-list/index.js

```diff
@@ -166,73 +158,50 @@ const TaskList = ( { overviewTasksVisibility, tasks } ) => {
 		return <div></div>;
 	}
 
-	const pendingTaskCount = visibleTasks.filter( ( task ) => ! task.completed )
-		.length;
 	return (
-		<Card
```

Comment:
Removed the Card, CardHeader and CardBody to allow this component to be placed inside a Card alongside the Welcome CardHeader